packages/backend-core/src/api/resources/User.ts

```diff
@@ -1,84 +1,79 @@
 import camelcaseKeys from 'camelcase-keys';
 
-import associationDefaults from '../utils/Associations';
-import filterKeys from '../utils/Filter';
 import { EmailAddress } from './EmailAddress';
 import { Association } from './Enums';
 import { ExternalAccount } from './ExternalAccount';
 import type { ExternalAccountJSON, UserJSON } from './JSON';
 import { PhoneNumber } from './PhoneNumber';
-import type { UserProps } from './Props';
+import type { EmailAddressProps, ExternalAccountProps, PhoneNumberProps, Web3WalletProps } from './Props';
 import { Web3Wallet } from './Web3Wallet';
 
-interface UserAssociations {
-  emailAddresses: EmailAddress[];
-  phoneNumbers: PhoneNumber[];
-  web3Wallets: Web3Wallet[];
-  externalAccounts: ExternalAccount[];
-}
-
-interface UserPayload extends UserProps, UserAssociations {}
-
-export interface User extends UserPayload {}
+export const userAssociations = {
```

Comment:
This is left here only until we decide if we want to keep this structure or not.

Reply:
A simpler modelling with just arrays would be better but this should be part of future work